mitmproxy/utils/arg_check.py

```diff
@@ -129,11 +131,15 @@ def check():
 
     for option in REPLACED.splitlines():
         if option in args:
+            if isinstance(REPLACEMENTS.get(option), list):
+                new_options = REPLACEMENTS.get(option)
+            else:
+                new_options = [REPLACEMENTS.get(option)]
             print(
                 "{} is deprecated.\n"
                 "Please use `{}` instead.".format(
                     option,
-                    REPLACEMENTS.get(option)
+                    "` or `".join(new_options)
```

Comment:
This fails for the existing options, see e.g. `" or ".join("--intercept")`.

Reply:
If there is only a single option as `str` in `REPLACEMENTS`, the string is put into a list in line 137.
I've tested it (now again) and it does work. Am I missing something?

```
❯ mitmproxy -i
usage: mitmproxy [options]
mitmproxy: error: unrecognized arguments: -i

-i is deprecated.
Please use `--intercept` instead.
```

Reply:
Yeah, I can't read.